docs/reference/modules/coordination.asciidoc

```diff
@@ -0,0 +1,461 @@
+
+[source,js]
+--------------------------------------------------
+# send the bootstrap document back to the cluster
+POST /_cluster/bootstrap
+{
+  "master_nodes_failure_tolerance": 1,
+  "master_nodes":[
+    {"id":"USpTGYaBSIKbgSUJR2Z9lg"},
+    {"id":"gSUJR2Z9lgUSpTGYaBSIKb"},
+    {"id":"2Z9lgUSpTgSUYaBSIKbJRG"}
+  ]
+}
+--------------------------------------------------
+// CONSOLE
+
+It is safe to repeatedly call `POST /_cluster/bootstrap`, and to call it on
+different nodes concurrently, but **it is vitally important** to use the same
+bootstrap document in each call.
+
+It is also possible to select the initial set of nodes in terms of their names
+rather than their IDs as follows.
+
+[source,js]
+--------------------------------------------------
+# send the bootstrap document back to the cluster
+POST /_cluster/bootstrap
+{
+  "master_nodes_failure_tolerance": 1,
+  "master_nodes":[
+    {"name":"master-a"},
+    {"name":"master-b"},
+    {"name":"master-c"}
+  ]
+}
+--------------------------------------------------
+// CONSOLE
+
+This can be useful if the node names are known (and known to be unique) in
+advance, and means that the first `GET /_cluster/bootstrap` call is not
+necessary. As above, it is safe to repeatedly call `POST /_cluster/bootstrap`,
+and to call it on different nodes concurrently, but **it is vitally important**
+to use the same bootstrap document in each call.
+
+[float]
+=== Manually-triggered elections
+
+It is possible to request that a particular node takes over from the elected
+master as follows:
+
+[source,js]
+--------------------------------------------------
+POST /_nodes/node_name/start_election
+--------------------------------------------------
+// CONSOLE
+
+Elections are not guaranteed to succeed, and a new leader may be elected at any
+time so even if this election does succeed then there may be another election,
+so there is no guarantee that the chosen node will be the elected master for
+any length of time.
+
+[float]
+=== Unsafe disaster recovery
+
+In a disaster situation a cluster may have lost half or more of its
+master-eligible nodes and therefore be in a state in which it cannot elect a
+master. There is no way to recover from this situation without risking data
+loss, but if there is no other viable path forwards then this may be necessary.
+This can be performed with the following command on a surviving node:
+
+[source,js]
+--------------------------------------------------
+POST /_nodes/_local/force_become_leader
+--------------------------------------------------
+// CONSOLE
+
+This works by reducing `cluster.master_nodes_failure_tolerance` to 0 and then
+forcibly overriding the current configuration with one in which the handling
+node is the only voting master, so that it forms a quorum on its own. Because
+there is a risk of data loss when performing this command it requires the
+`accept_data_loss` parameter to be set to `true` in the URL.
```

Comment:
add comment to say that `cluster.master_nodes_failure_tolerance` should be adjusted afterwards?

Reply:
Yep, done.